src/Security/Authentication/LoginManager.php

```diff
@@ -97,6 +104,8 @@ private function createToken(string $firewallName, UserInterface $user): TokenIn
 
     private function getUserChecker(FirewallConfig $config): UserCheckerInterface
     {
-        return UserCheckerClassResolver::resolveUserChecker($config->getUserChecker());
+        $userChecker = UserCheckerClassResolver::resolveUserChecker($config->getUserChecker());
```

Comment:
Here is something that works for me, but may be too hacky and rendundant 🤔  just to give you some input

```suggestion
        $firewallUserChecker = $config->getUserChecker();
        if (!$firewallUserChecker) {
            return $this->defaultUserChecker;
        }
        if (!class_exists($firewallUserChecker)) {
            $firewallUserChecker = '@' . $firewallUserChecker;
            $userChecker = UserCheckerClassResolver::resolveUserChecker($firewallUserChecker);
        } else {
            $userChecker = new $firewallUserChecker;
        }
        return $userChecker;
```
[](url)